docs-zh/faq/comparison.md

```diff
@@ -1,32 +1,35 @@
-# `envd` vs. Other tools
+# `envd` 和其他工具的比较
 
-## What is `envd`?
+## `envd`是什么?
 
-`envd` is a tool to build portable environments for machine learning and data science scenarios with Python-like syntax. 
+`envd`是一个为机器学习和数据科学场景建立可迁移，可复现环境的工具。
```

Comment:
Please add the blank between English and Chinese chars.

Reply:
fixed